src/main/java/com/google/plantasy/utils/GameUtils.java

```diff
@@ -63,16 +62,6 @@ public static boolean addUserToGame(String userId, Entity gameEntity, DatastoreS
     if(userId == ""){
       return false;
     }
-    if(gameEntity == null){
-        log.severe("found null game entity trying to user "+ userId +" to game");
-        return false;
-    }
-    if(datastore == null){
-        log.severe("found null datastore trying to game to user " + userId);
-        return false;
-    }
-
-    Game game = new Game(gameEntity);
```

Comment:
Does this run without this line (Game game = ...) ? We call game on line 69 but I don't think we define game in this function otherwise.

Reply:
I think I reverted some changes I shouldn't have thanks for catching that it all compiles and passes now.